VRDR/IJEMortality.cs

```diff
@@ -81,8 +81,16 @@ public IJEMortality(string ije)
                 // Grab the field value
                 string field = ije.Substring(info.Location - 1, info.Length);
                 // Set the value on this IJEMortality (and the embedded record)
```

Comment:
I apparently can't create a comment on a line higher than this, so I'm going to put it here. In regards to the "let's make validation optional", here is my proposal:

Add a second optional parameter to this constructor, say:
```
public IJEMortality(string ije, bool validate = true)
```
and wrap line 93 in an if, like:
```
if (validate) {
  this.Valid();
}
```

That way you can force this constructor to not validate if you provide the optional parameter as true, otherwise it will always run.